packages/synapse-react-client/src/synapse-queries/user/useCertificationQuiz.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  useMutation,
+  UseMutationOptions,
+  useQuery,
+  useQueryClient,
+  UseQueryOptions,
+} from 'react-query'
+import SynapseClient from '../../synapse-client'
+import { SynapseClientError } from '../../utils/SynapseClientError'
+import { useSynapseContext } from '../../utils/context'
+import { PassingRecord } from '@sage-bionetworks/synapse-types'
+import { QuizResponse } from '@sage-bionetworks/synapse-types'
+import { useGetCurrentUserProfile } from './useUserBundle'
+import { USER_BUNDLE_MASK_IS_CERTIFIED } from '../../utils/SynapseConstants'
+
+export function useGetPassingRecord(
+  userId: string = '',
+  options?: UseQueryOptions<PassingRecord, SynapseClientError>,
+) {
+  const { accessToken, keyFactory } = useSynapseContext()
+  return useQuery<PassingRecord, SynapseClientError>(
+    keyFactory.getPassingRecordQueryKey(userId),
+    () => SynapseClient.getPassingRecord(userId, accessToken),
+    options,
+  )
+}
+
+export function usePostCertifiedUserTestResponse(
+  options?: UseMutationOptions<PassingRecord, SynapseClientError, QuizResponse>,
+) {
+  const queryClient = useQueryClient()
+  const { accessToken, keyFactory } = useSynapseContext()
+  const { data: userProfile } = useGetCurrentUserProfile()
+
+  return useMutation<PassingRecord, SynapseClientError, QuizResponse>(
+    (quizResponse: QuizResponse) =>
+      SynapseClient.postCertifiedUserTestResponse(accessToken, quizResponse),
+    {
+      ...options,
+      onSuccess: async (updatedPassingRecord, variables, ctx) => {
```

Comment:
mutation handles clearing the react-query cache for the passing record and UserBundle IS_CERTIFIED mask.